src/relax/transform/fuse_ops.cc

```diff
@@ -1102,6 +1106,13 @@ class PatternBasedPartitioner : ExprVisitor {
       auto parent_group = GetGroupForBoundVar(binding->var);
       ICHECK(parent_group);
       parent_group->attrs.Set(attr::kComposite, pat_name_);
+      if (attrs_getter_ != nullptr) {
+        const auto& custom_attrs =
+            attrs_getter_(CreatePatternCheckContext(call, matches_opt.value()));
```

Comment:
A dup call to `CreatePatternCheckContext`

https://github.com/apache/tvm/blob/c3f6c07021d0b857d4340e7699f14dceba17b228/src/relax/transform/fuse_ops.cc#L1077

Reply:
Emm..... that can be simplified. It can be changed to: call the CreatePatternCheckContext once and use in both check && attrs_getter. That depends on comment 1, if PatternCheckContext can be used as the context for attrs_getter, then I'll change this part.

Reply:
Without seeing what `attrs_getter` does, it's hard to say if `CreatePatternCheckContext` is a reasonable source to query attributes from. That's why I'm asking for a test case.

Reply:
And I'll update the test case when the discussion about "context" is done. Currently adding the attributes is mainly for TensorRT translation. An "id_name" should be used to mark same expression in relax and TensorRT, so that each "layer" in the model can share their optimization strategy. The optimization here mainly refers to pruning, sparse and quantization.

Reply:
For example, I think you can recover the same information in the loop 
```
      for (const auto& [pat, match] : matches_opt.value()) {
```
using the `binding` argument to `VisitBinding_`.

Reply:
Em.... Let me add the test case to show why these attributes are needed